src/main/java/edu/kis/powp/jobs2d/command/visitor/CanvasFactory.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+public class CanvasFactory {
+    public static Canvas getCanvas(String canvasType){
+        if(canvasType == null)
+            return null;
+
+        if(canvasType.equalsIgnoreCase("A4"))
```

Comment:
You should use at least enum or constants (or provide separate factory methods).